js/Makefile

```diff
@@ -40,9 +40,21 @@ languages:
 engine:
 	cd engine; $(MAKE)
 
+.PHONY: pro-setup
+pro-setup:
+ifdef SEMGREP_PRO
+	ln -s -f ../_build ../_build
```

Comment:
what does this line do? it seems like a noop

Reply:
it's not, symlinks are weird and relative. So we're creating a symlink called `_build` in the parent directory, that will point to `_build` in the symlink's parent directory. So we're linking semgrep-pro's _build to semgrep/_build